api/pkg/db/migration/migration.go

```diff
@@ -36,9 +36,8 @@ func Migrate(api *app.APIBase) error {
 			addRefreshTokenChecksumColumnInUserTable(log),
 			updateCatalogBranchToMain(log),
 			addAvatarURLColumnInUsersTable(log),
-			removeCatgoryIdColumnAndConstraintsFromTagtable(log),
+			refreshAllTables(log),
 			updateResourcesCategoryTable(log),
-			removeExistingCategories(log),
```

Comment:
why are we removing migrations?

Reply:
don't we need to run them after refreshing the db?

Reply:
refreshAllTables this migration has to be run before all newly added migration, after that the remaning will be executed properly

Reply:
nope.... refresh_all_tables migration will do the job for us

Reply:
did you test this on a cluster?

Reply:
yes